<!--REQUEST_ID=bc1d9970-e3d6-4c8f-9091-2a55d820fb6c-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48718&attempt_id=fe8ede21-5d22-47a5-8a9e-5f32a94e5fea)

Attempted: anki

<details><summary>Partial log (click to expand)</summary><p>

```
vidix/pci_dev_ids.c:9:33: warning: 'dev_lst_001a' defined but not used [-Wunused-const-variable=]
 static const struct device_id_s dev_lst_001a[] = {
                                 ^~~~~~~~~~~~
vidix/pci_dev_ids.c:6:33: warning: 'dev_lst_0000' defined but not used [-Wunused-const-variable=]
 static const struct device_id_s dev_lst_0000[] = {
                                 ^~~~~~~~~~~~
builder for '/nix/store/0c2d13r5ahpn2hwpxp7ag4gk701q830l-mplayer-1.3.0.drv' failed with exit code 2
cannot build derivation '/nix/store/sal5vxfcxr4dm0mhvx9rp8mx6afaaqym-fix-paths.patch.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/4a3kj7zn5y1j03i3kvk9zq5i5n6m754c-anki-2.0.52.drv': 2 dependencies couldn't be built
error: build of '/nix/store/4a3kj7zn5y1j03i3kvk9zq5i5n6m754c-anki-2.0.52.drv' failed
```
</p></details>

--- a/pkgs/games/anki/default.nix
+++ b/pkgs/games/anki/default.nix
@@ -115,10 +115,8 @@ in buildPythonApplication rec {
     '';
 
     meta = with stdenv.lib; {
-      homepage = http://ankisrs.net/;
+      homepage = "https://apps.ankiweb.net/";
       description = "Spaced repetition flashcard program";
-      license = licenses.gpl3;
-
       longDescription = ''
         Anki is a program which makes remembering things easy. Because it is a lot
         more efficient than traditional study methods, you can either greatly
@@ -131,8 +129,9 @@ in buildPythonApplication rec {
         people's names and faces, brushing up on geography, mastering long poems,
         or even practicing guitar chords!
       '';
-
-      maintainers = with maintainers; [ the-kenny ];
+      license = licenses.agpl3Plus;
+      broken = stdenv.hostPlatform.isAarch64;
       platforms = platforms.mesaPlatforms;
+      maintainers = with maintainers; [ the-kenny ];
     };
 }

